packages/components/icons/README.md

```diff
@@ -105,3 +105,34 @@ const app = () => <LeadingIcon icon={<ExportIcon />} />;
 ### Where to use
 
 This component can be used wherever it is necessary to display a themed icon.
+
+## Custom Icon
+
+The custom icon is a an eye-catching visual element that should be used when an additional visual prominence is needed for displaying a custom SVG icon in the UI.
```

Comment:
I think we would need to be more specific about this component.

@FilPob could you help us coming up with another description which resembles what you think this component should be used for? 🙏 

Here is my attempt, in case it could be useful:
> This component is meant to be used whenever consumers need to render an icon which is not part of the [ui-kit icon set](https://uikit.commercetools.com/?path=/story/components-icons--all-icons). 
In order to keep visual consistency, we want to keep the available sizes of all icons equal. 
Bear in mind we would expect custom SVG icons to not contain size attributes so it can be controlled based on the components size attribute.

Reply:
Thanks @CarlosCortizasCT that sounds totally fine for me